public_dropin_environments/python3_onnx/Dockerfile

```diff
@@ -17,7 +17,7 @@ WORKDIR ${CODE_DIR}
 COPY ./*.sh ${CODE_DIR}/
 
 ENV WITH_ERROR_SERVER=1
-# Uncomment the following line to switch from Flask to uwsgi server
-#ENV PRODUCTION=1 MAX_WORKERS=1 SHOW_STACKTRACE=1
+# [DEPRECATED] Uncomment the following line to switch to produciton mode
```

Comment:
If its deprecated, why keep this line?

Reply:
Um :thinking: good question. I'll remove it from all related Dockerfiles